modules/coreos-layering-configuring-on.adoc

```diff
@@ -23,16 +37,41 @@ You need a separate `MachineOSConfig` CR for each machine config pool where you
 
 :FeatureName: On-cluster image layering
 include::snippets/technology-preview.adoc[]
+include::snippets//coreos-layering-configuring-on-pause.adoc[]
+
+In the case of a build failure, for example due to network issues or an invalid secret, the MCO retries the build three times before the job fails. The MCO creates a different build pod for each build attempt. You can use the build pod log to troubleshoot any build failures. 
+
+// Not in 4.18; maybe 4.19
+// You can manually rebuild your custom layered image by either modify your `MachineOSConfig` object or applying an annotation to the `MachineOSConfig` object as discussed in the following section.
+
+[discrete]
+[id="coreos-layering-configuring-on-limitations_{context}"]
+== On-cluster layering Technology Preview known limitations
+
+Note the following limitations when working with the on-cluster layering feature:
+
+* On-cluster layering is not supported on multi-architecture compute machines, single-node Openshift clusters, or disconnected clusters.
+* If you scale-up a machine set that uses a custom layered image, the nodes will reboot two times. The first, when the node is initially created with the base image and a second time when the custom layered image is applied.
+* Node disruption policies are not supported on nodes with a custom layered image. As a result the following configuration changes will cause a node reboot:
+** Changes to the configuration files in the `/var` or `/etc` directory
+** Adding or modifying a systemd service
+** Changing SSH keys
+** Removing mirroring rules from `ICSP`, `ITMS`, and `IDMS` objects
+** Changing the trusted CA, by updating user-ca-bundle configmap in the `openshift-config` namespace
+* The images used in creating custom layered images take up space in your push repository. Always be aware of the free space in your registry and prune the images as needed.
 
 .Prerequisites
 
-* You have enabled the `TechPreviewNoUpgrade` feature set by using the feature gates. For more information, see "Enabling features using feature gates".
+* You have enabled the `TechPreviewNoUpgrade` feature set by using the feature gates. For more information, see "Enabling features using feature gates". 
 
-* You have a copy of the global pull secret in the `openshift-machine-config-operator` namespace that the MCO needs in order to pull the base operating system image.
+* You have a copy of the pull secret in the `openshift-machine-config-operator` namespace that the MCO needs to pull the base operating system image. 
```

Comment:
This is no longer needed with the `v1` API. If a global pull secret is not provided in the MOSC, we will automatically grab the cluster global pull secret